custom-targets/util/clouddeploy/gcs.go

```diff
@@ -0,0 +1,226 @@
+// Copyright 2023 Google LLC
```

Comment:
Remove this file, the functionality in here was merged into clouddeploy.go. The functions were turned into methods on the request object to avoid having to pass the request object constantly.

Reply:
Done.